myscript.js

```diff
@@ -0,0 +1,14 @@
+$(document).ready(function(){
+
+    $("#firstname").html("Deepa");
+    $("#lastname").html("Natarajan !");
+    $(".mcc").css('color','blue');
+    $(".mcc").css('font-style','italic');
+    $("#hobby").append("<li>Painting</li>");
+    $("#hobby").append("<li>Playing with my Kid</li>");
```

Comment:
excellent job of using jQuery to target an element by ID and then append/alter content! You've also successfully added style for the mcc class! Well done!